aqueductcore/frontend/src/components/atoms/CheckboxField/index.tsx

```diff
@@ -0,0 +1,38 @@
+import { Box, Checkbox, Grid, styled } from "@mui/material";
+
+import { FieldDescription, FieldTitle, FieldType } from "components/atoms/sharedStyledComponents"
+
+const CheckboxInput = styled(Checkbox)`
+  padding: ${({ theme }) => theme.spacing(0.5)});
+  border-color: ${({ theme }) => theme.palette.mode === "dark" ? theme.palette.common.white : theme.palette.grey[600]};
+`;
+
+interface CheckboxProps {
+  title: string,
+  field: string,
+  description: string,
+  defaultValue?: string;
+}
```

Comment:
I guess we can also use the same pattern here.
```suggestion
interface CheckboxProps extends PluginFieldBase{
  defaultValue?: string;
}
```

Reply:
Yeah, missed this.